crates/test_utils/tests/connectors/selenium.rs

```diff
@@ -812,8 +812,14 @@ pub fn should_ignore_test(name: &str) -> bool {
     let tests_to_ignore: HashSet<String> =
         serde_json::from_value(conf).unwrap_or_else(|_| HashSet::new());
     let modules: Vec<_> = name.split("::").collect();
-    let file_match = format!("{}::*", <&str>::clone(&modules[1]));
-    let module_name = modules[1..3].join("::");
+    let file_match = format!(
+        "{}::*",
+        <&str>::clone(modules.get(1).expect("Error fetching filed module"))
+    );
+    let module_name = modules
+        .get(1..3)
+        .expect("Error fetching filed module name")
+        .join("::");
```

Comment:
Would these error messages help? You can ignore this suggestion if not.

```suggestion
    let file_match = format!(
        "{}::*",
        <&str>::clone(modules.get(1).expect("Error obtaining module path segment"))
    );
    let module_name = modules
        .get(1..3)
        .expect("Error obtaining module path segment")
        .join("::");
```